teams.md

```diff
@@ -0,0 +1,208 @@
+# Teams
+
+This document describes the responsibilities, permissions and criteria for membership
+for each of the teams associated with the _pandas_ project.
+
```

Comment:
Is it worth summarising at the top that there are 10 teams.

- Core library team
- pandas-stubs team
- Finance team
- Infrastructure team
- Documentation team
- Code of conduct team
- Triage team
- Contributor community team
- Website team
- Outreach team

Is there sufficient differentiation and lack of overlap in the last four teams to warrant individual teams for those? Maybe the tasks are actually quite distinct but the titles naively suggest some crossovers.

Is it worth adding that explicitly individuals can, and are, expected in some cases to be members of multiple teams.

Can I also one question about the documentation team. Since the permissions are those of a subset of core and presumably core team also has interest in maintaining documentation is there, do people transition from docs team to core team or can they remain in both. Is there any benefit or difference for a core member also being in the docs team

Reply:
Thanks, I raised the point about possible overlap, as I imagine having more teams introduces more complexity of organisation, maintenance, coordination, business continuity management, etc, so it could be worth the disadvantage of adding associated repsonsibilities to one team for the advantage of reducing team numbers.

With BCM in mind are teams of one permissible? For example I would expect the finance team should comprise at least two people if not three to minimise the risk of oversight, especially if they are ones with access and visibility over the funds. Similary for the Outreach team, should community passwords control be managed by at least two people.

Reply:
> With BCM in mind are teams of one permissible? For example I would expect the finance team should comprise at least two people if not three to minimise the risk of oversight, especially if they are ones with access and visibility over the funds. Similary for the Outreach team, should community passwords control be managed by at least two people.

I don't think we want any teams of one.  It may be worthwhile for us to add minimums (and maximums, where appropriate) for each team

Reply:
if we have a minimum defined and that is not achieved, do the responsibilities of that team revert to the core (library) team by default until a new team structure is defined (i.e. governance documents updated)?

Reply:
> if we have a minimum defined and that is not achieved, do the responsibilities of that team revert to the core (library) team by default until a new team structure is defined (i.e. governance documents updated)?

That would depend on the team.   To be discussed.

Reply:
> > Is there sufficient differentiation and lack of overlap in the last four teams to warrant individual teams for those?
> 
> .... Website team is responsible for the website. Outreach team is responsible for communications/announcements of new versions via email, Twitter/X, etc.

I think there could be potential overlap between website and outreach (as website can also be considered as a public communication channel, or at least the content of the website), but that will also depend on the actual people working on it if they would want to combine it or not.

Which makes me wonder: for this initial version of the listing of teams, should we maybe limit ourselves more to the teams that already exist somewhat, or use the full listing of teams we would like to have (and which we would certainly approve as a new team).

Because for the same example of website and outreach (or also documentation), I would really love it if we would actually have a team for those topics, but that is not really the case right now.

Reply:
I've added the list at the top, as suggested, clarified that people can be on more than one team, and that if it is difficult to find enough people on a team, the Steering Committee can decide to assign  the responsibilities of the underserved team to another team.  All in next revision.